pkg/skel/skel.go

```diff
@@ -319,8 +364,8 @@ func PluginMainWithError(cmdAdd, cmdCheck, cmdDel func(_ *CmdArgs) error, versio
 // as JSON to stdout and call os.Exit(1).
 //
 // To have more control over error handling, use PluginMainWithError() instead.
-func PluginMain(cmdAdd, cmdCheck, cmdDel func(_ *CmdArgs) error, versionInfo version.PluginInfo, about string) {
-	if e := PluginMainWithError(cmdAdd, cmdCheck, cmdDel, versionInfo, about); e != nil {
+func PluginMain(cmdAdd, cmdCheck, cmdDel, cmdGC func(_ *CmdArgs) error, versionInfo version.PluginInfo, about string) {
```

Comment:
Rather than adding more functions here, let's just deprecate the existing functions and leave their signatures as-is.